src/main/java/me/ryanhamshire/GriefPrevention/DataStore.java

```diff
@@ -1166,7 +1166,7 @@ void resizeClaimWithChecks(Player player, PlayerData playerData, int newx1, int
             }
             catch (ArithmeticException e)
             {
-                GriefPrevention.sendMessage(player, TextMode.Err, Messages.ResizeClaimInsufficientArea, String.valueOf(GriefPrevention.instance.config_claims_minArea));
+                GriefPrevention.sendMessage(player, TextMode.Err, Messages.ResizeNeedMoreBlocks, String.valueOf(Integer.MAX_VALUE));
```

Comment:
Was this catch logic always incorrect?

Reply:
Yeah, it was added pretty recently and I think we were more focused on it preventing issues than presentation. I believe this message is a bit more user-friendly.